pkg/project/types.go

```diff
@@ -17,6 +17,7 @@ type ProjectStatus struct {
 	Active bool `json:"active"`
 }
 
+// ProjectList holds a list of project
```

Comment:
Since `Project` is a type, would `ProjectList holds a list of Project` make more sense?

Reply:
done